app/src/main/res/values-ru/sura_names.xml

```diff
@@ -70,50 +70,169 @@
     <item>Аль-Мульк</item>
     <item>Аль-Калам</item>
     <item>Аль-Хакка</item>
-    <item>Аль-Мааридж</item>
+    <item>Аль-Ма\'аридж</item>
     <item>Нух</item>
     <item>Аль-Джинн</item>
     <item>Аль-Муззаммиль</item>
-    <item>Аль-Муддассир</item>
+    <item>Аль-Мудассир</item>
     <item>Аль-Кийама</item>
     <item>Аль-Инсан</item>
-    <item>Аль-Мурсалат</item>
-    <item>Ан-Наба</item>
-    <item>Ан-Назиат</item>
-    <item>Абаса</item>
+    <item>Аль-Мурсалят</item>
+    <item>Ан-Наба\'</item>
+    <item>Ан-Нази\'ат</item>
+    <item>\'Абаса</item>
     <item>Ат-Таквир</item>
     <item>Аль-Инфитар</item>
     <item>Аль-Мутаффифин</item>
     <item>Аль-Иншикак</item>
     <item>Аль-Бурудж</item>
     <item>Ат-Тарик</item>
-    <item>Аль-Аля</item>
+    <item>Аль-А\'ла</item>
     <item>Аль-Гашийа</item>
     <item>Аль-Фаджр</item>
     <item>Аль-Балад</item>
     <item>Аш-Шамс</item>
-    <item>Аль-Лейл</item>
+    <item>Аль-Лайл</item>
     <item>Ад-Духа</item>
-    <item>Аль-Инширах</item>
+    <item>Аш-Шарх</item>
     <item>Ат-Тин</item>
     <item>Аль-Алак</item>
     <item>Аль-Кадр</item>
-    <item>Аль-Беййина</item>
+    <item>Аль-Баййина</item>
     <item>Аз-Залзала</item>
     <item>Аль-Адийат</item>
-    <item>Аль-Кариа</item>
+    <item>Аль-Кари\'а</item>
     <item>Ат-Такасур</item>
-    <item>Аль-Аср</item>
+    <item>Аль-\'Аср</item>
     <item>Аль-Хумаза</item>
     <item>Аль-Филь</item>
-    <item>Курейш</item>
-    <item>Аль-Маун</item>
+    <item>Курайш</item>
+    <item>Аль-Ма\'ун</item>
     <item>Аль-Каусар</item>
     <item>Аль-Кафирун</item>
     <item>Ан-Наср</item>
     <item>Аль-Масад</item>
     <item>Аль-Ихлас</item>
-    <item>Аль-Фаляк</item>
+    <item>Аль-Фалак</item>
     <item>Ан-Нас</item>
   </string-array>
+
+  <!-- If sura name has no translation, please add empty double quotes to the
```

Comment:
any better ideas? i just leave it blank, but i know it shows it as a warning. also worried that it will give you back "" literally - i am guessing it doesn't though?